### PR TITLE
Read_me file updated according to given format.

### PR DESCRIPTION
**Pull request by ayush461**
## Related Issue 

- Updating the read_me file according to new given format.

Closes: #60

### Describe the changes you've made

I changed the structure of the read_me file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,222 +1,134 @@
...
-<a href="https://github.com/praveenscience/Internship-LMS-FrontEnd/graphs/contributors">
-  <img src="https://contrib.rocks/image?repo=praveenscience/Internship-LMS-FrontEnd" />
-</a>
-
-
--->
-
-# DevTernShip: Internship OnBoarding Platform
  
+ ## DevTernShip: Internship OnBoarding Platform
```
Comment (praveenscience) [inline README.md:2]: Don't leave any trailing spaces before and after.

File: README.md [hunk 2]
```diff
@@ -1,222 +1,134 @@
...
  
+ ## DevTernShip: Internship OnBoarding Platform 
 
 ![](https://user-images.githubusercontent.com/1830380/111150758-65810c00-8586-11eb-8941-7985fb66f23c.png)
 
-# BADGES
 
+[![Issues](https://img.shields.io/github/issues/praveenscience/Internship-LMS-FrontEnd)](https://github.com/praveenscience/Internship-LMS-FrontEnd/issues)
+[![PRs](https://img.shields.io/github/issues-pr/praveenscience/Internship-LMS-FrontEnd)](https://github.com/praveenscience/Internship-LMS-FrontEnd/pulls)
+[![Maintenance](https://img.shields.io/maintenance/yes/2020?color=green&logo=github)](https://github.com/praveenscience/)
```
Comment (praveenscience) [inline README.md:9]: Should be:
```suggestion
[![Maintenance](https://img.shields.io/maintenance/yes/2021?color=green&logo=github)](https://github.com/praveenscience/)
```

File: README.md [hunk 3]
```diff
@@ -1,222 +1,134 @@
...
+[![Forks](https://img.shields.io/github/forks/praveenscience/Internship-LMS-FrontEnd?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd) 
+[![Stars](https://img.shields.io/github/stars/praveenscience/Internship-LMS-FrontEnd?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd) 
+[![Watchers](https://img.shields.io/github/watchers/praveenscience/foss-events?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd)
```
Comment (praveenscience) [inline README.md:12]: Dei, why da?
```suggestion
[![Watchers](https://img.shields.io/github/watchers/praveenscience/Internship-LMS-FrontEnd?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd)
```

File: README.md [hunk 4]
```diff
@@ -1,222 +1,134 @@
...
-
- # Motivation
-
-
-
+ We are in the mission of solving a crisis for a lot of small and medium enterprises, who are trying to recruit interns for their projects. This platform aims to onboard new interns, who have applied to the companies and help them understand the process and make the transition smooth.
 
- # Methodology
-As this project is not your usual Open Source project, this is not going be a bug fixing or a robotic one. Please read the messages carefully and let's work as a team. We will be following Agile Methodology, specifically [Spotify Agile Model](https://rb.gy/tzvtsj) with some [Coding Standards](https://rb.gy/jmo3xp) being followed, which I'll discuss later.
+- Motivation
```
Comment (praveenscience) [inline README.md:18]: Should be a Level 3 Heading.

File: README.md [hunk 5]
```diff
@@ -1,222 +1,134 @@
...
-# BADGES
 
+[![Issues](https://img.shields.io/github/issues/praveenscience/Internship-LMS-FrontEnd)](https://github.com/praveenscience/Internship-LMS-FrontEnd/issues)
+[![PRs](https://img.shields.io/github/issues-pr/praveenscience/Internship-LMS-FrontEnd)](https://github.com/praveenscience/Internship-LMS-FrontEnd/pulls)
+[![Maintenance](https://img.shields.io/maintenance/yes/2020?color=green&logo=github)](https://github.com/praveenscience/)
+[![Forks](https://img.shields.io/github/forks/praveenscience/Internship-LMS-FrontEnd?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd) 
+[![Stars](https://img.shields.io/github/stars/praveenscience/Internship-LMS-FrontEnd?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd) 
+[![Watchers](https://img.shields.io/github/watchers/praveenscience/foss-events?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd)
 
 # Introduction
```
Comment (praveenscience) [inline README.md:14]: Should be a Level 2 Heading.

File: README.md [hunk 6]
```diff
@@ -1,222 +1,134 @@
...
 
- 
-
-
- # Motivation
-
-
-
+ We are in the mission of solving a crisis for a lot of small and medium enterprises, who are trying to recruit interns for their projects. This platform aims to onboard new interns, who have applied to the companies and help them understand the process and make the transition smooth.
```
Comment (praveenscience) [inline README.md:16]: Why space in front? Let's rephrase this for a better intro or should we give it to someone else?

File: README.md [hunk 7]
```diff
@@ -1,222 +1,134 @@
...
 
- # Methodology
-As this project is not your usual Open Source project, this is not going be a bug fixing or a robotic one. Please read the messages carefully and let's work as a team. We will be following Agile Methodology, specifically [Spotify Agile Model](https://rb.gy/tzvtsj) with some [Coding Standards](https://rb.gy/jmo3xp) being followed, which I'll discuss later.
+- Motivation
+  - Atuomate the process of regular tasks.
```
Comment (praveenscience) [inline README.md:19]: I am gonna kill you for this.

File: README.md [hunk 8]
```diff
@@ -1,222 +1,134 @@
...
+  - Make the task easy and stress free of managment members.
+  - Save the time of people in doing same work multiple times.
 
+- Methodology
```
Comment (praveenscience) [inline README.md:23]: Should be a Level 3 Heading.

File: README.md [hunk 9]
```diff
@@ -1,222 +1,134 @@
...
 
+- Video Links
```
Comment (praveenscience) [inline README.md:25]: Should be a Level 3 Heading.

File: README.md [hunk 10]
```diff
@@ -1,222 +1,134 @@
...
-
-
-
-# Resources
-
-OpenSource Intro (https://rb.gy/dxnv5y)
-30 Days of React (Event: https://rb.gy/zsgyja, PlayList: https://rb.gy/cxk3ek)
-
-
+- Resources
```
Comment (praveenscience) [inline README.md:31]: We'll do something about this, you fix the current one. 😅

File: README.md [hunk 11]
```diff
@@ -1,222 +1,134 @@
...
+  - [OpenSource Intro](https://rb.gy/dxnv5y)
+  - 30 Days of React
+    - [Event](https://rb.gy/zsgyja)
```
Comment (praveenscience) [inline README.md:34]: Could be Event Page or Event Landing Page?

File: README.md [hunk 12]
```diff
@@ -1,222 +1,134 @@
...
+    - [PlayList](https://rb.gy/cxk3ek)
 
 # GSSoC Participation
+![](https://i.imgur.com/BR9Q5Pd.png)
```
Comment (praveenscience) [inline README.md:38]: Give some space dude. Make it Level 2 heading. Only one level 1 heading should be there.

File: README.md [hunk 13]
```diff
@@ -1,222 +1,134 @@
...
+  - Rules of GSSoC 21
+  - Levels & Points
```
Comment (praveenscience) [inline README.md:40]: Make these Level 3 Heading. Leave it for others.

File: README.md [hunk 14]
```diff
@@ -1,222 +1,134 @@
...
+    <td>Level 2</td>
+    <td>15</td>
+    
+  </tr>
+  <tr>
+    <td>Level 3</td>
+    <td>25</td>
+    
+  </tr>
+</table>
```
Comment (praveenscience) [inline README.md:67]: Remove all the empty lines here. Check if these scores are right. Where did you get them from?

File: README.md [hunk 15]
```diff
@@ -1,222 +1,134 @@
...
 
-GSSoC Logo (https://i.imgur.com/BR9Q5Pd.png)
 
-Rules of GSSoC 21
-Levels & Points
+# Product Information
+  - Screenshots
```
Comment (praveenscience) [inline README.md:71]: Make these two Level 2 and Level 3 Heading.

File: README.md [hunk 16]
```diff
@@ -1,222 +1,134 @@
...
 
 
 
-# Product Information
+### Tech Stack
 
-Screenshots
+<img src="https://img.shields.io/badge/html5%20-%23E34F26.svg?&style=for-the-badge&logo=html5&logoColor=white"/> <img src="https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white"/>  <img src="https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E"/>
+<img src="https://img.shields.io/badge/node.js%20-%2343853D.svg?&style=for-the-badge&logo=node.js&logoColor=white"/>   <img src="https://img.shields.io/badge/github%20-%23121011.svg?&style=for-the-badge&logo=github&logoColor=white"/> <img src="https://img.shields.io/badge/heroku%20-%23430098.svg?&style=for-the-badge&logo=heroku&logoColor=white"/> <img src="https://img.shields.io/badge/express.js%20-%23404d59.svg?&style=for-the-badge"/> <img src ="https://img.shields.io/badge/MongoDB-%234ea94b.svg?&style=for-the-badge&logo=mongodb&logoColor=white"/>
```
Comment (praveenscience) [inline README.md:78]: Can we use Markdown for these, please?

File: README.md [hunk 17]
```diff
@@ -1,222 +1,134 @@
...
 
-* Bootstrap 5 for the Front End
-* React JS with both Class & Functional Components
-* Node JS & Express JS
-* MySQL or PostgreSQL (yet to be decided)
+Deployment
+Husky 
+Netlify/Heroku
+Github Actions
+</pre>
```
Comment (praveenscience) [inline README.md:101]: Again, make it look good, just use normal simple Markdown Lists.

File: README.md [hunk 18]
```diff
@@ -1,222 +1,134 @@
...
-
-
-
-
 # Installation & Running
 
-
-
-
 # Team
```
Comment (praveenscience) [inline README.md:109]: Level 2 Heading Please.

File: README.md [hunk 19]
```diff
@@ -1,222 +1,134 @@
...
 
 Admin & Mentors
 
 * Praveen Kumar Purushothaman ([Website](https://praveen.science/) &#1421; [LinkedIn](https://www.linkedin.com/in/praveentech/) &#1421; [YouTube](https://www.youtube.com/praveenscience?sub_confirmation=1))
-* Manju Ramachandran ([LinkedIn](https://www.linkedin.com/in/manjuramachandran10/))
-* Saloni Sawarkar ([LinkedIn](https://www.linkedin.com/in/saloni-sawarkar/))
-
```
Comment (praveenscience) [inline README.md:114]: No need of Line Space.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ayush461) [reply README.md:2]: ok sir

**Pull request closed without merging**